doc/examples/applications/plot_3d_image_processing.py

```diff
@@ -0,0 +1,311 @@
+                       [0, 1, 1],
+                       [0, 1, 0]]])
+    verts = verts * (60, 256, 256)
+    verts += [i, 0, 0]
+
+    ax.add_collection3d(
+        Poly3DCollection(
+            verts,
+            facecolors="magenta",
+            linewidths=1,
+            edgecolors="black"
+        )
+    )
+
+    ax.set_xlabel("plane")
+    ax.set_xlim(0, 100)
+    ax.set_ylabel("row")
+    ax.set_zlabel("col")
+
+    # Autoscale plot axes
+    scaling = np.array([getattr(ax,
+                                "get_{}lim".format(dim))() for dim in "xyz"])
+    ax.auto_scale_xyz(* [[np.min(scaling), np.max(scaling)]] * 3)
+
+
+def explore_slices(data, cmap="gray"):
+    from ipywidgets import interact
+    N = len(data)
+
+    @interact(plane=(0, N - 1))
+    def display_slice(plane=34):
+        fig, ax = plt.subplots(figsize=(20, 5))
+
+        ax_3D = fig.add_subplot(133, projection="3d")
+
+        show_plane(ax, data[plane], title="Plane {}".format(plane), cmap=cmap)
+        slice_in_3D(ax_3D, plane)
+
+        plt.show()
+
+    return display_slice
+
+
+explore_slices(data);
+
+#####################################################################
+# Adjust exposure
+# ===============
+# Scikit-image's `exposure` module contains a number of functions for
+# adjusting image contrast. These functions operate on pixel values.
+# Generally, image dimensionality or pixel spacing needn't be considered.
+
+#####################################################################
+# `Gamma correction <https://en.wikipedia.org/wiki/Gamma_correction>`_
+# brightens or darkens an image. A power-law transform, where `gamma` denotes
+# the power-law exponent, is applied to each pixel in the image: `gamma < 1`
+# will brighten an image, while `gamma > 1` will darken an image.
+
+
+def plot_hist(ax, data, title=None):
+    # Helper function for plotting histograms
+    ax.hist(data.ravel(), bins=256)
+    ax.ticklabel_format(axis="y", style="scientific", scilimits=(0, 0))
+
+    if title:
+        ax.set_title(title)
+
+
+gamma_low_val = 0.5
+gamma_low = exposure.adjust_gamma(data, gamma=gamma_low_val)
+
+gamma_high_val = 1.5
+gamma_high = exposure.adjust_gamma(data, gamma=gamma_high_val)
+
+_, ((a, b, c), (d, e, f)) = plt.subplots(nrows=2, ncols=3, figsize=(12, 8))
+
+show_plane(a, data[32], title="Original")
+show_plane(b, gamma_low[32], title="Gamma = {}".format(gamma_low_val))
+show_plane(c, gamma_high[32], title="Gamma = {}".format(gamma_high_val))
+
+plot_hist(d, data)
+plot_hist(e, gamma_low)
+plot_hist(f, gamma_high)
+# sphinx_gallery_thumbnail_number = 4
+
+#####################################################################
+# `Histogram
+# equalization <https://en.wikipedia.org/wiki/Histogram_equalization>`_
+# improves contrast in an image by redistributing pixel intensities. The most
+# common pixel intensities get spread out, letting areas of lower local
+# contrast gain a higher contrast. This may enhance background noise though.
```

Comment:
```suggestion
# common pixel intensities get spread out, increasing contrast in low-contrast
# areas. One downside of this approach is that it may enhance background noise.
```